library/src/main/scala/java/nio/TypedArrayBuffer.scala

```diff
@@ -0,0 +1,53 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js API               **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-lang.org/     **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+package java.nio
```

Comment:
OK I'll move these three classes to `scala.scalajs.typedarray`. And for the bridge to be able to access the methods it calls from `java.nio`, I'll make them public.